src/middlewares/authorization.js

```diff
@@ -0,0 +1,23 @@
+import decodeJwt from '../helpers/tokens/decode.token';
```

Comment:
we now have the authorization middleware of @MCFrank16 
We should use it I think

Reply:
Thanks, let me switch @nkpremices